src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -498,7 +498,7 @@ function mightCauseHorizontallyOverlappingBubbles(element) {
     //we can be very conservative and say that if the text
     //box isn't taking up the whole width, it *might* cause
     //an overlap
-    if(element.hasClass('bloom-alwaysShowBubble')) {
+    if((element.hasClass) && element.hasClass('bloom-alwaysShowBubble')) {
```

Comment:
I've fixed the bug in my latest PR (since I added this bit to mightCauseHorizontallyOverlappingBubbles()) both by making sure all callers pass the right thing AND by using $(element).hasClass instead of the bare element.hasClass. The bare form worked when I passed a JQuery item and not when the other caller passed an HTMLElement. (Oh the benefits of TypeScript!)